cli/towerSysUtil.ml

```diff
@@ -72,7 +77,8 @@ let in_dir path f =
 
 let foldp_paths f p acc dir =
   let dh = Unix.opendir dir in
-  let files = in_dir dir (fun () -> all_files "" [] dh) in
+  let full_files = in_dir dir (fun () -> all_files "" [] dh) in
+  let files = List.filter (fun file_name -> not (ignorable_file file_name)) full_files in
   let () = Unix.closedir dh in
   List.fold_left (fun acc file ->
     if p file dir then f acc file else acc
```

Comment:
I think `ignorable_file` should be used by the caller of `foldp_paths` in the `p` (predicate) parameter which is used here to filter the paths while they are being folded.